### PR TITLE
[ta] Fix Test Automation #5

### PR DESCRIPTION
**Pull request by jecfish**
- Fix wrong url and typo. It has to be "code coverage" because this article is about code coverage. In the article, we have a section specifically explain code coverage != test coverage.
- Fix low contrast table view in dark mode. The previous hardcoded value only work in light mode. Update it to use existing variable.
- Fix list image padding issue. Previously, the list item in "There are five branches in the code example:" format incorrectly due to the inline image. 
- Fix branch coverage code highlight. Previously not highlighted at all.

### CODE REVIEW

File: src/site/content/en/blog/ta-code-coverage/index.md
Comment (jecfish) [inline src/site/content/en/blog/ta-code-coverage/index.md:163]: purposely use text instead of js here and other places to show the highlighted code more, especially in dark mode.

**Review by addyosmani — APPROVED** [review]
LGTM

**Comment by jecfish** [comment]
To whoever has permission to approve, please merge this after approval .

**Review by heyawhite — APPROVED** [review]

**Pull request merged**
